packages/@ember/-internals/routing/lib/system/router.ts

```diff
@@ -706,7 +712,7 @@ class EmberRouter extends EmberObject.extend(Evented) implements Evented {
     }
   }
 
-  replaceWith(...args: any[]) {
+  replaceWith(...args: RouteArgs): Transition {
```

Comment:
Should this have docs?

Reply:
I think *yes and they should mark it as deprecated, same as `transitionTo`*.

Reply:
`Router#transitionTo` isn't marked as deprecated.